client/app/lib/onboarding.coffee

```diff
@@ -207,7 +207,8 @@ module.exports = class Onboarding
 
     # Trigger a 'done' pseudo-event, corresponding to onboarding end.
     triggerDone: ()->
-        throw new Error 'Not implemented yet'
+        # redirection to the cozy home
+        window.location.replace '/'
```

Comment:
I think that onboarding should not be related to `window` object in any way.

My view :
A listener object like `Application` can subscribe to the "done" pseudo-event like it does for "stepChanged", so we need to add a method `onboarding.onDone(callback)`, which register a callback. Then, `triggerDone` just call the callback.
This way, this is the Application object which knows what to do when the onboarding it's done. It can redirect the brower to a configured url.